rel/files/ejabberd.cfg

```diff
@@ -885,34 +885,36 @@
 %   {muc, [{host, "muc.@HOST@"}]}
 % ]}
 
-  %%
-  %% Configuration for Amazon SNS notifications.
-  %%
-
-%  {mod_aws_sns, [
-    %% AWS credentials, region and host configuration
-%    {access_key_id, "AKIAJAZYHOIPY6A2PESA"},
-%    {secret_access_key, "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"},
-%    {region, "eu-west-1"},
-%    {account_id, "251423380551"},
-%    {region, "eu-west-1"},
-%    {sns_host, "sns.eu-west-1.amazonaws.com"},
-
-    %% Messages from this MUC host will be sent to the SNS topic
-%    {muc_host, "muc.@HOST@"},
-
-    %% Plugin module for defining custom message attributes and user identification
-%    {plugin_module, mod_aws_sns_defaults},
-
-    %% Topic name configurations. Removing topic will disable this specific SNS notification
-%    {presence_updates_topic, "user_presence_updated-dev-1"},  %% For presence updates
-%    {pm_messages_topic, "user_message_sent-dev-1"},           %% For private chat messages
-%    {muc_messages_topic, "user_messagegroup_sent-dev-1"}      %% For group chat messages
-
-    %% Pool options
-%    {pool_size, 100}, %% Worker pool size for publishing notifications
-%    {publish_retry_count, 2}, %% Retry count in case of publish error
-%    {publish_retry_time_ms, 50} %% Base exponential backoff time (in ms) for publish errors
+% {mod_event_pusher, [
+%   {backends, [
+%     %%
+%     %% Configuration for Amazon SNS notifications.
+%     %%
+%     {sns, [
+%       %% AWS credentials, region and host configuration
+%       {access_key_id, "AKIAJAZYHOIPY6A2PESA"},
+%       {secret_access_key, "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"},
+%       {region, "eu-west-1"},
+%       {account_id, "251423380551"},
+%       {region, "eu-west-1"},
+%       {sns_host, "sns.eu-west-1.amazonaws.com"},
+%
+%       %% Messages from this MUC host will be sent to the SNS topic
+%       {muc_host, "muc.@HOST@"},
+%
+%       %% Plugin module for defining custom message attributes and user identification
+%       {plugin_module, mod_event_pusher_sns_defaults},
+%
+%       %% Topic name configurations. Removing topic will disable this specific SNS notification
```

Comment:
opic name configurations. Removing the topic will disable this specific SNS notification